packages/orchestrator-ui-components/src/components/WfoPydanticForm/fields/WfoText.tsx

```diff
@@ -18,6 +18,7 @@ export const Text: PydanticFormControlledElement = ({
 
     return (
         <EuiFieldText
+            data-testid="text-field"
```

Comment:
use formfield.id like the others?
```suggestion
            data-testid={pydanticFormField.id}
```